tutorials/CHT/flow-over-plate/buoyantPimpleFoam-laplacianFoam/precice-config.xml

```diff
@@ -48,13 +48,13 @@
         <exchange data="Heat-Flux" mesh="Solid-Mesh" from="Solid" to="Fluid"/>
         <relative-convergence-measure limit="1.0e-6" data="Temperature" mesh="Fluid-Mesh"/>
         <extrapolation-order value="2"/>
-        <post-processing:IQN-ILS>
+        <acceleration:IQN-ILS>
```

Comment:
This corrensponds to a diferent change in preCICE, but since in the tutorials repository they are treated as one, it's ok to treat it as one here as well.

Reply:
However, we need to update this for every tutorial.

Reply:
https://github.com/precice/tutorials/pull/60 already contains this change. E.g. [here](https://github.com/precice/tutorials/pull/60/files#diff-850e1be720a942764ab616149b633307R50-R56). Sorry, if I am causing confusion. I thought that https://github.com/precice/openfoam-adapter/pull/107 is just the OpenFOAM version of https://github.com/precice/tutorials/pull/60. (And it is more convenient for our current testing problems, if the breaking changes are not scattered)

Reply:
Yes, but we have more tutorials here (i.e. the nearest-projection variant). Just let me take care of this.

Reply:
> And it is more convenient for our current testing problems, if the breaking changes are not scattered

I think I can hack around this problem in my testing branch (see https://github.com/precice/systemtests/commit/2460f9afeae217eca4e21ae3b8ccba819d706229). Therefore, you can ignore this argument. If you want to keep things separated, feel free to `git revert` 1e826a8